openedx_proversity_reports/edxapp_wrapper/get_modulestore.py

```diff
@@ -10,3 +10,11 @@ def get_modulestore(*args, **kwargs):
     backend = import_module(backend_function)
 
     return backend.get_modulestore_backend(*args, **kwargs)
+
+def item_not_found_error():
+    """ Get the ItemNotFoundError exception. """
```

Comment:
.

Reply:
We are using the first person convention in this file, so it's not neccesary to change.